controllers/api/opennebula/machines.py

```diff
@@ -27,13 +27,16 @@ def index(self):
     @cherrypy.tools.isAuthorised(redirect=True)
     @cherrypy.tools.jinja(template="machines/history.html")
     def history(self):
-        pass
+        CLOUDPLATFORM = cherrypy.request.config.get("cloudPlatform")
```

Comment:
why does `history` just return a dict with the the cloud platform in it?

Reply:
and the same thing happens under both OpenStack and OpenNebula?

Reply:
The dict is used for jinja templating variables in the html page for history. I decided to just use the same system as the other pages for consistency even if there is only one variable that gets used.

Reply:
This was fixed "offline" to this discussion thread, for both OpenStack and Open Nebula.